recipes/colcon-common-extensions/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "colcon-common-extensions" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/colcon/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a8d78f5f01f99d1d6f0bf490427281773dab7ff046ba4e746c80816c9419dda6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
```

Comment:
This syntax can only be used if the package is noarch python. If it’s not you have to use skip true py<35 way.

Reply:
To add, the issue with doing it this way is that you have one build for each supported python version and python =2.7 conflicts with python >=3.5.